pkg/jobrunaggregator/jobrunaggregatoranalyzer/junit.go

```diff
@@ -187,14 +190,22 @@ func aggregateTestCase(combined *junit.TestCase, jobName, toAddJobRunID string,
 	switch {
 	case toAdd.FailureOutput != nil:
 		humanURL := jobrunaggregatorapi.GetHumanURL(jobName, toAddJobRunID)
+		testCaseFailure := TestCaseFailure{
+			JobRunID:       toAddJobRunID,
+			HumanURL:       humanURL,
+			GCSArtifactURL: jobrunaggregatorapi.GetGCSArtifactURL(jobName, toAddJobRunID),
+		}
+		if jobrunaggregatorlib.IsDisruptionTest(toAdd.Name) {
+			disruptionSeconds, err := jobrunaggregatorlib.GetOutageSecondsFromMessage(toAdd.FailureOutput.Message)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to get disruption seconds from %q\n", toAdd.FailureOutput.Message)
+			} else {
+				testCaseFailure.DisruptionSeconds = disruptionSeconds
```

Comment:
we can succeed and have more than zero disruptionseconds.  I think that needs to be reflected here as well.